longevity_test.py

```diff
@@ -110,6 +110,9 @@ def run_prepare_write_cmd(self):
             self._pre_create_schema(keyspace_num, scylla_encryption_options=self.params.get(
                 'scylla_encryption_options', None))
 
+        if not prepare_write_cmd:
```

Comment:
not sure why but it doesn't look that safe to me.
Trying to think on test scenarios that we may have prepare_write_cmd or variations of that.

Reply:
Go through our tests, the only such configuration with "pre_create_keyspace" and without "prepare_write_cmd" we have on counters_multidc_test. 
May be it wil be safer to add prepare_write_cmd for that test? What do you think @roydahan 
@bentsi @fruch  what is your opinion?

Reply:
Don't think this is safe too and I don't understand where the motivation comes from.
Anyway, you are changing the logic here, and this can be dangerous: you run the  `_pre_create_schema` and `pre_create_keyspace` independently of the `prepare_write_cmd` param though I think it could be reasonable.
If it is needed you should move the pre_create_schema and pre_create_keyspace outside of the `run_prepare_write_cmd` to make them independent from the `run_prepare_write_cmd` method.

Reply:
Moved to 3 different functions.

Job is successfully running: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/abykov/job/longevity-counters-multidc-test/13/